packages/manager/apps/pci-load-balancer/src/api/hook/useGateways.ts

```diff
@@ -20,3 +29,63 @@ export const useGetSubnetGateways = (
     enabled: !!projectId && !!regionName && !!subnetId,
     throwOnError: true,
   });
+
+export const useRegionPlans = (regionName: string) => {
```

Comment:
This function is really restrictive with this output, could you make it more generic ? or move the parameter out of the hook